_im-plugin/ism/api-config-notice.md

```diff
@@ -0,0 +1,168 @@
+---
+layout: default
+title: Configuring ISM notifications settings
```

Comment:
This feature has no dependency on ISM. The title should be "Configuring long-running index operation notifications settings" or "Configuring index operation notifications" for short. And its doc parent should be "MANAGING INDEXES".